src/main/java/de/unistuttgart/iaas/amyassist/amy/plugin/spotify/PlayerLogic.java

```diff
@@ -0,0 +1,423 @@
+/*
+ * Amy Assist
+ *
+ * Personal Assistance System
+ *
+ * @author Tim Neumann, Leon Kiefer, Benno Krauss, Christian Braeuner, Felix Burk, Florian Bauer, Kai Menzel, Lars Buttgereit, Muhammed Kaya, Patrick Gebhardt, Patrick Singer, Tobias Siemonsen
+ *
+ */
+package de.unistuttgart.iaas.amyassist.amy.plugin.spotify;
+
+import java.io.IOException;
+import java.net.URI;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+import com.google.gson.JsonParser;
+import com.wrapper.spotify.exceptions.SpotifyWebApiException;
+import com.wrapper.spotify.model_objects.miscellaneous.Device;
+import com.wrapper.spotify.requests.data.player.GetUsersAvailableDevicesRequest;
+import com.wrapper.spotify.requests.data.player.PauseUsersPlaybackRequest;
+import com.wrapper.spotify.requests.data.player.SetVolumeForUsersPlaybackRequest;
+import com.wrapper.spotify.requests.data.player.SkipUsersPlaybackToNextTrackRequest;
+import com.wrapper.spotify.requests.data.player.SkipUsersPlaybackToPreviousTrackRequest;
+import com.wrapper.spotify.requests.data.player.StartResumeUsersPlaybackRequest;
+
+import de.unistuttgart.iaas.amyassist.amy.core.di.annotation.Service;
+import de.unistuttgart.iaas.amyassist.amy.core.plugin.api.Init;
+
+/**
+ * TODO: This class have methods to control a spotify client from a user. For examlpe play, pause playback or search for music tracks etc.
+ * 
+ * @author Lars Buttgereit
+ */
+@Service(PlayerLogic.class)
+public class PlayerLogic {
+	private Authorization auth;
+	private String deviceID = null;
+	private Search search;
+	private int volume = 50;
+	// private String deviceName = null;
+	private ArrayList<String[]> actualSearchResult = null;
+	
+	public PlayerLogic() {
+		init();
+	}
+
+	@Init
```

Comment:
Remove the Annotation its deprecated, it have no effect because the method is called from the constructor